cylc/flow/scheduler.py

```diff
@@ -744,25 +756,41 @@ def restart_remote_init(self):
             ):
                 distinct_install_target_platforms.append(itask.platform)
 
-        incomplete_init = False
         for platform in distinct_install_target_platforms:
+            # skip remote init for localhost
+            install_target = platform['install target']
+            if install_target == get_localhost_install_target():
+                continue
+            # set off remote init
             self.task_job_mgr.task_remote_mgr.remote_init(
                 platform, self.server.curve_auth,
                 self.server.client_pub_key_dir)
-            status = self.task_job_mgr.task_remote_mgr.remote_init_map[
-                platform['install target']]
-            if status in (REMOTE_INIT_IN_PROGRESS,
-                          REMOTE_FILE_INSTALL_IN_PROGRESS):
-                incomplete_init = True
-                break
-            if status == REMOTE_INIT_DONE:
-                self.task_job_mgr.task_remote_mgr.file_install(platform)
-        if incomplete_init:
-            # TODO: Review whether this sleep is needed.
-            sleep(1.0)
             # Remote init/file-install is done via process pool
             self.proc_pool.process()
-        self.command_poll_tasks(['*/*'])
+            # add platform to map (to be picked up on main loop)
+            self.incomplete_ri_map[install_target] = platform
+
+    def manage_remote_init(self, platform):
+        """Manage the remote init/file install process for restarts.
+            Called within the main loop.
+            Starts file installation when Remote init is complete.
+            Removes complete installations or installations encountering SSH
+            error (remote init will take place on next job submission).
+        """
+        install_target = platform['install target']
+        status = self.task_job_mgr.task_remote_mgr.remote_init_map[
+            install_target]
+        if status == REMOTE_INIT_DONE:
+            LOG.info(
```

Comment:
I think that this should be at `DEBUG` level.

This feels like a weird thing to say here - I'd expect this message to come from the start of the code doing re-installation, not from the tail of the initialization - might be better to say
```
"Init complete: Ready to start remote file installation for "
```

Reply:
I think it's ok at the INFO level, but could do with being the same message as we would raise for a regular remote-init / file-install:

https://github.com/cylc/cylc-flow/blob/32d263396f937585869344d7be91cee320cc98de/cylc/flow/task_remote_mgr.py#L251

https://github.com/cylc/cylc-flow/blob/32d263396f937585869344d7be91cee320cc98de/cylc/flow/task_remote_mgr.py#L504